src/Microsoft.MobileBlazorBindings/ShellNavigation/KnownIssues.md

```diff
@@ -0,0 +1,36 @@
+﻿# Known issues with ShellNavigationManager
```

Comment:
Thank you for writing this up! Should we track all of this as a new issue in the repo? I fear that if it's in an MD file in the code it'll never be seen again.

Reply:
I agree, Github issue created.